src/toil/batchSystems/kubernetes.py

```diff
@@ -599,24 +622,35 @@ def _isPodStuckOOM(self, podObject, minFreeBytes=1024 * 1024 * 2):
 
         We assume the pod has only one container, as Toil's pods do.
 
+        If the metrics service is not working, we treat the pod as not being
+        stuck OOM.
```

Comment:
Because if we treated jobs as stuck when the metrics service wasn't there, we would treat all jobs as stuck if the cluster didn't have a metrics service installed and working. Sometimes it goes down, some people never install it, and on the Toil managed clusters the cluster won't scale out just to give the metrics service a place to run, so it doesn't initially come up until the workflow makes the cluster scale out.

Reply:
I've added a bit more explanation.